app/src/main/java/ai/elimu/appstore/dao/CustomDaoMaster.java

```diff
@@ -41,6 +41,11 @@ public void onUpgrade(Database db, int oldVersion, int newVersion) {
             if (oldVersion < 2000003) {
                 DbMigrationHelper.migrate(db, ApplicationVersionDao.class);
             }
+
+            if (oldVersion < 2000010) {
+                dropAllTables(db, true);
```

Comment:
Why is the entire database deleted when upgrading to this version?

Reply:
The database structure has changed due to 1 new column: checkSumMd5.
We have 2 options here: 
1. Re-creating database to adopt new table schema. (current option)
2. Alter the current schema (we don't have to drop current databases)

Do you want option 2 instead?